liberbatch.py

```diff
@@ -1,19 +1,15 @@
 import os, meichecker, liberupdatev5
+PATH = "your path here"
 
-directory = os.fsencode("C:\\Users\\cole_\\Downloads\\liberUsualis - Copy\\liber")
+directory = os.fsencode(PATH)
 err=""
 
-i = 0 #for debugging
 for mefile in os.listdir(directory):
     filename = os.fsdecode(mefile)
     if filename.endswith("corr.mei"):
-        liberupdate.main(filename)
+        liberupdatev5.main(filename)
         print(filename +" has been updated")
         err= err+ (meichecker.main(filename[:-4]+"NEW2.mei")+"\n")

```

Comment:
For something like this, a linter will generally prefer an f-string. 

```suggestion
        err += f"{meichecker.main(filename[:-4]}NEW2.mei)\n")
```